### ISSUE TITLE
Remove docs/containers for insecure releases

### ISSUE THREAD

**Issue by phlax** [envoyproxy/envoy#16034]
### description

checking the docs page, i think some of the docs relate to builds/releases for which there are known sec issues:

https://www.envoyproxy.io/docs

(also, some of the links are dead)

afaict there are also still dockerhub images with a similar issue:

https://hub.docker.com/r/envoyproxy/envoy/tags?page=1&ordering=last_updated&name=1.17

if you read the docs and look at the published images there is no way of knowing that some versions should be avoided

imho we should probably just delete all images and docs other than the latest for each minor version

**Comment by phlax** [comment]
cc @mattklein123

**Comment by mattklein123** [comment]
I don't think we can remove old images, because for better or worse people may be pinned to them.

I think removing old releases is very reasonable. Do you want to do a PR over on the docs site? https://github.com/envoyproxy/envoyproxy.github.io/blob/c470b304486a3ca1edbc50527c1f5ea32e2db290/_config.yml#L25

**Comment by phlax** [comment]
sort of yes - the repo is huge - need to find some disk space to do it

after playing with netlify over on envoy-playground - i like it a lot - but i guess that is a separate issue

**Comment by phlax** [comment]
also just thinking we may need to change envoy ci too to prevent changes being overwritten (ill check further)

**Comment by phlax** [comment]
the other thing i need to procede with this are:

- supported versions
- list of known insecure versions
